homeassistant/components/zha/core/const.py

```diff
@@ -22,6 +22,7 @@
 from homeassistant.components.lock import DOMAIN as LOCK
 from homeassistant.components.number import DOMAIN as NUMBER
 from homeassistant.components.sensor import DOMAIN as SENSOR
+from homeassistant.components.siren import DOMAIN as SIREN
```

Comment:
For future PR, import `Platform` from `homeassistant.const` and use `Platform.SIREN` etc.

Reply:
Will clean these up in the near future

Reply:
PR is here: https://github.com/home-assistant/core/pull/61016 

NOTE: Leaving it in draft until after the beta to avoid conflicts if we have to patch something in ZHA